src/rtc/rtc_s1.py

```diff
@@ -257,6 +258,46 @@ def process_child_runconfig(path_runconfig_burst,
     return result_child_process
 
 
+def process_child_runconfig_subprocess(path_runconfig_burst,
+                            path_burst_logfile=None,
+                            flag_full_logfile_format=None,
+                            keep_burst_runconfig=False):
+    '''
+    single worker to process runconfig from terminal using `subprocess`
+    Parameters
+    ----------
+    path_runconfig_burst: str
+        Path to the burst runconfig
+    path_burst_logfile: str
+        Path to the burst logfile
+    full_log_format: bool
+        Enable full formatting of log messages.
+        See `get_rtc_s1_parser()`
+    keep_burst_runconfig: bool
+        Keep the child runconfig when `True`;
+        delete it after done with the processing when `False`
+    Returns
+    -------
+    result_child_process: int
+        0 when the child process has completed succesfully
+    '''
+    os.environ['OMP_NUM_THREADS'] = "1"
```

Comment:
I wanted to limit the threads available for the child processes. In case of `geocode()`, it can be done by specifying `memory_mode`, but not sure if `topo()` also has something similar feature.
Note that we run `topo()` when computing layover shadow mask.

Reply:
I'm not sure if this is a good idea. Is this what is causing your code to break in aurora? Otherwise, maybe we can leave it as is for now. We need to review the ISCE3 code that is computing the layover/shadow mask anyway.

Reply:
Unfortunately, aurora was not able to deal with 28 * `OMP_NUM_THREADS` of processes when computing the layover shadow mask in the child processes. I would prefer to suppress the number of threads in `topo()` level, which does not seem to be available (correct me if I'm wrong).